src/ipvs/ip_vs_conhash.c

```diff
@@ -222,10 +228,13 @@ static int dp_vs_conhash_del_dest(struct dp_vs_service *svc,
                 inet_addr_equal(dest->af, &p_conhash_node->addr, &dest->addr) &&
                 p_conhash_node->port == dest->port) {
             p_node = &(p_conhash_node->node);
-            ret = conhash_del_node(p_sched_data->conhash, p_node);
-            if (ret < 0) {
-                RTE_LOG(ERR, SERVICE, "%s: conhash_del_node failed\n", __func__);
-                return EDPVS_INVAL;
+            if (p_conhash_node->weight) {
+                ret = conhash_del_node(p_sched_data->conhash, p_node);
+                if (ret < 0) {
+                    RTE_LOG(ERR, SERVICE, "%s: conhash_del_node failed\n",
+                            __func__);
+                    return EDPVS_INVAL;
+                }
```

Comment:
if `p_conhash_node->weight==0`,  `node_fini(p_node)` is called without `conhash_del_node`. Is this OK?

Reply:
it's ok because `node_fini` will be callled in the following line (line 239)